tests/scripts/minikube.sh

```diff
@@ -63,6 +63,15 @@ case "${1:-}" in
     echo "updating the rook images"
     copy_image_to_cluster ${BUILD_REGISTRY}/rook-amd64 rook/rook:master
     copy_image_to_cluster ${BUILD_REGISTRY}/toolbox-amd64 rook/toolbox:master
+    # If current kubectl context is minikube, restart rook pods
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "minikube" ];
+    then
+        echo "Restarting Rook pods in \"rook\" namespace."
```

Comment:
how about updating the rook-operator and rook-agents image?

Reply:
Good point, added pods restarts in rook-system namespace.